platform/ios/docs/guides/For Style Authors.md

```diff
@@ -41,7 +41,7 @@ Studio, especially when multitasking is enabled. Your user’s viewing distance
 may be shorter than on a desktop computer. Some of your users may use the Larger
 Dynamic Type and Accessibility Text features to increase the size of all text on
 the device. You can use the
-[runtime styling API](#manipulating-the-style-at-runtime) to adjust your style’s
+[runtime styling API](https://www.mapbox.com/ios-sdk/maps/overview/runtime-styling/) to adjust your style’s
```

Comment:
This change is unnecessary; there’s still a “Manipulating the style at runtime” section. That’s important because it’s the only documentation we have between /ios-sdk and the jazzy docset that names the specific classes that form the runtime styling API.